### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by sofiane-soufi** [swellstores/community#95] [Help] [answered]
Hi, 

I'm getting a 500 error trying to access my store:

"Your dev storefront encountered an error. See application logs in the storefront editor or contact your developer for help."

https://teeth-whitening.swell.store/

Any idea where it can come from?

Regards,

**Comment by cristianstanescu** [comment, marked as answer]
Thanks for raising this @sofiane-soufi. While I can't replicate the error now and I see your storefront working, I'll make sure we'll keep an eye on it and investigate further.

Please share any other related information that could help us identify the problem and improve our system.
